gspread/utils.py

```diff
@@ -249,7 +249,7 @@ def a1_to_rowcol(label):
 
         col = 0
         for i, c in enumerate(reversed(column_label)):
-            col += (ord(c) - MAGIC_NUMBER) * (26 ** i)
+            col += (ord(c) - MAGIC_NUMBER) * (26**i)
```

Comment:
This is your issue with the linter.

use the following command to format your code (you can refer to the [CONTRIBUTION_GUIDELINES](https://github.com/burnash/gspread/blob/master/.github/CONTRIBUTING.md)):
`tox -e format` it will format your code so it will pass the linter.

Reply:
I don't get it, using `black` give me that using `tox -e format` also give me that. No difference doing in venv
I think I have to drop out from this

Reply:
Don't worry mate. we can solve this later.

Keep your PR open and let's focus on the design you choose. I ça handle the format later.